pkg/virtual/workspaces/cmd/cmd.go

```diff
@@ -148,3 +149,7 @@ func (o *WorkspacesSubCommandOptions) PrepareVirtualWorkspaces() ([]rootapiserve
 	}
 	return informerStarts, virtualWorkspaces, nil
 }
+
+func (o *WorkspacesSubCommandOptions) GetCurrentKubeContext() string {
```

Comment:
Why does _this_ part of the command get to choose what the current context is? When can this vary?

Reply:
Typically, the *personal workspaces* command (`virtual-workspaces workspaces`) publishes 2 contexts:
- one (with the ./.../personal URL suffix) for the personal workspaces view
- one (with the ./.../organization URL suffix) for the organization workspaces view

But typically the `personal` view is the preferred one, or the default one, so it will be made the current context in the generated kubeconfig.